### ISSUE TITLE
Calibration upload a63f62

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#9]
Calibration images were uploaded to the directory that starts with “`a63f62_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

**Comment by tmc-au** [comment]
I am the uploader of the Tamron 28-300 RAW files.
By the way - the link to the the work-low description is throwing up a 404.

**Comment by bronger** [comment]
Link fixed.  Thanks for the heads-up!

**Comment by junkyardsparkle** [comment]
@tmc-au: This is a nice batch of calibration images in general, but I'm having a little trouble with the TCA correction because of noise. If you could take one more at  each of the same focal lengths (28, 35, 50, 70, 97, 188, 300) at base ISO, I think it would help.

@bronger: Possibly a note in the tutorial about not using high ISO for TCA images would be good?

**Comment by tmc-au** [comment]
Weather and light conditions permitting, it will take and upload new images over the weekend. What is the best way to deliver them ?

**Comment by bronger** [comment]
ISO 1000 is much for calibration shots but on the other hand, I can see the colour fringes clearly.  Have you tried the `--simple-tca` option to `calibrate.py`?  If `tca_correct` has problems – which sometimes happens – you have to measure by hand with Darktable.  The values become the `vr` and `vb` parameters in the XML. You need a little bit of experience to do that, so if you have problems, just say so, then I do TCA.

**Comment by bronger** [comment]
Also note that Lensfun cannot remove LCA, only TCA.  LCA is often called “purple fringing” but it can also be green (seldomly).  It is not easy to distinguish.  TCA always comes in pairs: One side of the bright stripe is greenish, the other purpleish.  In contrast, LCA is a single fringe.

**Comment by junkyardsparkle** [comment]
As it happens, I do have plenty of experience with hand-tuning tca in darktable; I'll give that a try, if the `--simple-tca` option doesn't work... give me 24 hours, it's midnight here now...

I wasn't hoping to eliminate the purple fringing, it just seemed like the corrections for many focal lengths were only about half as strong as they should be on the actual  TCA. I'm not familiar with the internals of `tca_correct`, but I was guessing the noise wasn't helping matters, equating to a lack of sharpness, more or less.

**Comment by junkyardsparkle** [comment]
Ok, after getting similar results from the `--simple-tca` option, I made an attempt at hand-tuning via darktable, especially the longer focal lengths (you can easily find the "artificial" values by looking for lots of trailing zeros). In some cases, I found that correcting for one corner made things worse in another, and I now suspect this is why the "compromise" generated results seemed like under-corrections to me (sorry for blaming you, high ISO noise!) and, indeed, a similar compromise is the best I could do by hand in those cases. Call it a learning experience...

**Comment by bronger** [comment]
So, we keep the calculated TCA values?  We could change that even after having closed it, but I assume that what's in Git right now is what you think is least bad.

**Comment by bronger** [comment]
About the lens model name: This numeric name is not final and actually a sign that I should file a bug report against exiv2.  In this case, it seems to be correct with newer exiv2 versions (and Exiftool, for that matter).

**Comment by junkyardsparkle** [comment]
Essentially, I started the manual TCA correction at the longer focal lengths, which were showing substantial improvement, but as I encountered  the "conflicting corners" issue (look at `TAMRON_28-300mm_F__3.5-6.3_Di_VC_PZD_A010--050mm--14.0_50mm-f14-2.CR2` as an example, comparing correction for upper left and lower right corners) and I wasn't able to do much better than `tca_correct` after that (although I did change a few that seemed like at least slight overall improvements). The focus distance for some of these is given as ~5-6m... that might be a factor? So yes, what I commited seemed like the best overall to me based on those images, but it might benefit from further investigating with different eyes/photos.

Hopefully @tmc-au can give the profile some testing and report any problem areas. I wasn't sure if using the numeric model ID in the database made sense even as an interim measure, but it does make testing easier.
